src/webgl/p5.Framebuffer.js

```diff
@@ -394,7 +399,7 @@ class Framebuffer {
       gl.bindRenderbuffer(gl.RENDERBUFFER, this.colorRenderbuffer);
       gl.renderbufferStorageMultisample(
         gl.RENDERBUFFER,
-        Math.min(2, gl.getParameter(gl.MAX_SAMPLES)),
+        Math.min(this.antialiasSamples, gl.getParameter(gl.MAX_SAMPLES)),
```

Comment:
To be extra safe should we wrap this in a Math.max 0 to prevent negative values? I'm not sure if webGL would just treat it as a 0 if it was negative anyway or not but better safe than sorry!

`Math.max(0, Math.min(this.antialiasSamples, gl.getParameter(gl.MAX_SAMPLES)));`